VMBackup/main/backuplogger.py

```diff
@@ -50,7 +50,8 @@ def log(self, msg, local=False, level='Info'):
                 log_msg = self.log_to_con_py3(msg, level)
             else:
                 log_msg = "{0}  {1}  {2} \n".format(str(datetime.datetime.now()) , level , msg)
-                self.log_to_con(log_msg)
+                if(self.enforced_local_flag_value != False):
+                    self.log_to_con(log_msg)
```

Comment:
was this also provided as part of private fix? looks like this could be reason for resolving the issue as well.

Reply:
No this was not provided as private fix